src/drawer.js

```diff
@@ -220,7 +220,23 @@ export default class Drawer extends util.Observer {
      * @return {number}
      */
     getScrollX() {
-        return Math.round(this.wrapper.scrollLeft * this.params.pixelRatio);
+        const pixelRatio = this.params.pixelRatio;
+        let x = Math.round(this.wrapper.scrollLeft * pixelRatio);
+
+        // In cases of elastic scroll (safari with mouse wheel) you can
+        // scroll beyond the limits of the container
+        // Calculate and floor the scrollable extent to make sure an out
+        // of bounds value is not returned
+        // https://github.com/katspaugh/wavesurfer.js/issues/1312
```

Comment:
you dont have to include the full url, just a ticket reference should be fine.

Reply:
Gotcha, will update.